automated-comments/csharp/general/convert_field_to_const.md

```diff
@@ -0,0 +1 @@
+Consider converting the `%{name}` field to a `const`, as the value is intended never to change. Using constants is not only more common, there are also some subtle differences between a `const` and a `field`, which are explained in [this StackOverflow post](https://stackoverflow.com/questions/755685/static-readonly-vs-const#755693).
```

Comment:
Will the analyser still _block_ if this change is not made? Or is this always a "tip"? If it still blocks:

```suggestion
Convert the `%{name}` field to a `const`, as the value is intended never to change. Using constants is not only more common, there are also some subtle differences between a `const` and a `field`, which are explained in [this StackOverflow post](https://stackoverflow.com/questions/755685/static-readonly-vs-const#755693).
```

Reply:
The analyser will not block (this is used in the approve_with_comment) path.